### ISSUE TITLE
No exp on kill

### ISSUE THREAD

**Issue by deadspidervenom** [TheFlash787/Mobpression#13]
**Describe the bug**
When killing a stacked mob example "2x creeper", no exp is dropped.

**Expected behavior**
I expected exp to drop for the mob

**Screenshots**
n/a

**Plugin Version:**
 Mobpression 1.0.4-SNAPSHOT

**Additional context**

**Comment by TheFlash787** [comment]
Hey!

Sorry about the delay.

When killing this creeper, did it drop any items along with it? (as it should do)

Thank you!

**Comment by deadspidervenom** [comment]
yes it dropped items like it should, but exp was not dropped until it was
just a normal creeper

**Comment by Mango-Ataulfo** [comment]
Hola, también los animales si le das de comer, solo puede comer uno sin importar si son varios, y por ejemplo si tienes 30 ovejas en uno y en otro tienes 40 ovejas y se unen te da dos, la única solución que encontré fue poner una valla para que cada animal este separado y poder multiplicar sin unirse para no perder la cantidad de animales ya acumulados, y para los mob agregarlos a la lista negra para que no se junten y se pierda la experiencia, no se si puede sugerir si puedes crear una herramienta que se pueda fabricar tipo no se una cinta adhesiva para poder comprimir a los animales en un cierto radio creo que eso puede solucionar mientras el problema de los animales para que se compriman solos. Gracias por leerme y crear este plugin genial

**Comment by TheFlash787** [comment]
> 
> 
> Hola, también los animales si le das de comer, solo puede comer uno sin importar si son varios, y por ejemplo si tienes 30 ovejas en uno y en otro tienes 40 ovejas y se unen te da dos, la única solución que encontré fue poner una valla para que cada animal este separado y poder multiplicar sin unirse para no perder la cantidad de animales ya acumulados, y para los mob agregarlos a la lista negra para que no se junten y se pierda la experiencia, no se si puede sugerir si puedes crear una herramienta que se pueda fabricar tipo no se una cinta adhesiva para poder comprimir a los animales en un cierto radio creo que eso puede solucionar mientras el problema de los animales para que se compriman solos. Gracias por leerme y crear este plugin genial

@Mango-Ataulfo Hey! This is unrelated to the initial post but I understand the issue. Regarding mob farms and such, this plugin may not work well for the time being but you can whitelist certain mobs from compressing which will be a temporary fix for the time being.

I'll try to keep you posted.

**Comment by Alfonso87639** [comment]
> **Describe the bug**
> When killing a stacked mob example "2x creeper", no exp is dropped.
> 
> **Expected behavior**
> I expected exp to drop for the mob
> 
> **Screenshots**
> n/a
> 
> **Plugin Version:**
> Mobpression 1.0.4-SNAPSHOT
> 
> **Additional context**

I have the same issue in the same plugin version, is there a solution? 
In 2x or more mobs compressed, it drops the item, but not the xp
